lib/events/dynamoevents/dynamoevents.go

```diff
@@ -1251,7 +1274,12 @@ func (l *Log) migrateMatchingEvents(ctx context.Context, filterExpr string, tran
 	var startKey map[string]*dynamodb.AttributeValue
 	workerCounter := atomic.NewInt32(0)
 	totalProcessed := atomic.NewInt32(0)
-	workerErrors := make(chan error, maxMigrationWorkers)
```

Comment:
is `maxMigrationWorkers` still used anywhere? Should we have an upper limit?

Reply:
It's now used as an upper bound on the calculation.